lib/pf/UnifiedApi/Controller/Config/TrafficShapings.pm

```diff
@@ -0,0 +1,59 @@
+package pf::UnifiedApi::Controller::Config::TrafficShapings;
```

Comment:
Would be better as traffics_shapping as the plural form IMO

Reply:
traffic is an uncountable noun. So there is no plural form.
Also the word 'traffics' is a verb.

Reply:
So maybe:
traffic_shaping_policy + traffic_shaping_policies ?

Reply:
I like _policy / _policies